### PR TITLE
Feature/navigation mobile

### PR DESCRIPTION
**Pull request by mfbarrera**
Navigation in mobile, Logo, menu and overlay created, Links and icons added.

### CODE REVIEW

File: src/data/data.js [hunk 1]
```diff
@@ -1,6 +1,54 @@
...
+            text: 'eventos',
+            url: 'https://www.google.com',
+          },
+          {
+            text: 'fundación',
+            url: 'https://www.google.comm',
+          },
+          {
+            text: 'contacto',
+            url: 'https://www.google.comm',
```
Comment (caritonaranjo) [inline src/data/data.js]: please remove the, extra  M  in the .com

File: src/js/components/navigation.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  menuButton.addEventListener('click',()=>{
+    overlay.classList.add('nav__overlay--active')
+  })
+  overlayButton.addEventListener('click', ()=>{
+    overlay.classList.remove('nav__overlay--active')
+  })
+  return header;
+}
+
+export { navigationComponent }
```
Comment (caritonaranjo) [inline src/js/components/navigation.js]: leave an empty line at the end.

File: src/js/components/navigation.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  const imageLogo = newContainer('img');
+  imageLogo.src = data.image.src;
+  imageLogo.alt = data.image.alt;
+
+  const logoLink = newContainer('a',[imageLogo],['nav__image']) 
+  logoLink.href = data.logoUrl;
+
+  const imageMenu = newContainer('img')
+  imageMenu.src = 'menu.png';
+  imageMenu.alt = 'menu icon';
```
Comment (caritonaranjo) [inline src/js/components/navigation.js:32]: load this information from the data

File: src/js/components/navigation.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+  const menuButton = newContainer('button',[imageMenu],['nav__icon'])
+  
+  const collapsedMenuContainer = newContainer('div',[logoLink, menuButton],['nav__main'])
+  // Overlay
+  const imageClose = newContainer('img');
+  imageClose.src = 'close.png';
+  imageClose.alt = 'close button';
```
Comment (caritonaranjo) [inline src/js/components/navigation.js:40]: load this information from the data

**Review by caritonaranjo — CHANGES_REQUESTED** [review]
looking good but some nits are requiered

File: src/data/data.js (hunk 1 above)
Comment (mfbarrera) [reply src/data/data.js]: Done

File: src/js/components/navigation.js (hunk 3 above)
Comment (mfbarrera) [reply src/js/components/navigation.js:32]: it is coming from the assets not need to be added in the data

File: src/js/components/navigation.js (hunk 2 above)
Comment (mfbarrera) [reply src/js/components/navigation.js]: done

File: src/js/components/navigation.js (hunk 4 above)
Comment (mfbarrera) [reply src/js/components/navigation.js:40]: it is coming from the assets not need to be added in the data

**Review by valuarte — COMMENTED** [review]
navigation fer
Hi, Fer. Please double-check the design, because the top and bottom padding doesn't match with the design. Check the screenshot.

**Review by caritonaranjo — APPROVED** [review]
great work!!!

**Comment by mfbarrera** [comment]
@valuarte Changes are done, please validate! thank you

**Review by valuarte — APPROVED** [review]
Great job Fer.

**Pull request merged**